tools/_python_framework/requirements.txt

```diff
@@ -0,0 +1,4 @@
+crypto
```

Comment:
I'm making sure... as it isn't the first time you'd try to push something not tested.